From 3534539a25ce3c43c8abe9c44e2df96369c0860d Mon Sep 17 00:00:00 2001
From: Tianfei zhang <tianfei.zhang@intel.com>
Date: Mon, 8 Mar 2021 07:05:12 -0500
Subject: [PATCH 45/93] fpga: m10bmc-sec: Fix FPGA_M10_BMC_SEC_UPDATE
 dependency

FPGA_M10_BMC_SEC_UPDATE has a dependency on MFD_INTEL_M10_BMC
which has been renamed to MFD_INTEL_M10_BMC_CORE.

Signed-off-by: Tianfei zhang <tianfei.zhang@intel.com>
---
 drivers/fpga/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/fpga/Kconfig b/drivers/fpga/Kconfig
index 71515c626997..9ca6c1d596e9 100644
--- a/drivers/fpga/Kconfig
+++ b/drivers/fpga/Kconfig
@@ -254,9 +254,9 @@ config FPGA_MGR_VERSAL_FPGA
 
 config FPGA_M10_BMC_SEC_UPDATE
 	tristate "Intel MAX10 BMC Secure Update driver"
-	depends on MFD_INTEL_M10_BMC
+	depends on MFD_INTEL_M10_BMC_CORE
 	select FW_LOADER
 	select FW_UPLOAD
 	help
 	  Secure update support for the Intel MAX10 board management
 	  controller.
-- 
2.39.0

